pkgs/development/compilers/openjdk/openjfx/17/default.nix

```
@@ -48,6 +48,9 @@ in stdenv.mkDerivation {
       -i modules/javafx.web/src/main/native/Source/bmalloc/bmalloc/Heap.cpp \
          modules/javafx.web/src/main/native/Source/bmalloc/bmalloc/IsoSharedPageInlines.h
 
+    substituteInPlace modules/javafx.web/src/main/native/Source/JavaScriptCore/offlineasm/parser.rb \
+      --replace "File.exists?" "File.exist?"
```

```suggestion
      --replace-fail "File.exists?" "File.exist?"
```

--replace is deprecated and replaced with more specific flags